### PR TITLE
Increase hollow-node qps limit

### PR DESCRIPTION
**Pull request by gmarek**
@saad-ali - test only change. I really need tests to be faster...

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37223)

### CODE REVIEW

**Comment by dims** [comment]
LGTM 👍

File: cmd/kubemark/hollow-node.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func (c *HollowNodeConfig) createClientConfigFromFile() (*restclient.Config, err
 		return nil, fmt.Errorf("error while creating kubeconfig: %v", err)
 	}
 	config.ContentType = c.ContentType
+	config.QPS = 10
```
Comment (wojtek-t) [inline cmd/kubemark/hollow-node.go]: You need to also bump burst.

Comment (gmarek) [inline cmd/kubemark/hollow-node.go]: We don't need to, but I'm happy to do it:)

**Comment by saad-ali** [comment]
> @saad-ali - test only change. I really need tests to be faster...

Ack.

**Pull request merged**